tests/manage/mcg/ui/test_mcg_ui.py

```diff
@@ -240,20 +240,30 @@ def teardown(self):
 
     @ui
     @tier1
-    @skipif_ocs_version("!=4.8")
     @pytest.mark.parametrize(
-        argnames=["storageclass", "bucketclass"],
+        argnames=["storageclass", "bucketclass", "delete_via"],
```

Comment:
since the scenario in 2097772 is  covered, please also add the Bugzilla marker

Reply:
referring to your prev comment, it will be covered in separate PR